client/index.html

```diff
@@ -7,6 +7,17 @@
 
   <body>
     <div id="app"/>
+    <nav class="navigation-bar">
+      <div id="top">
+        <ul class="nav-bar">
+          <li id="homepage-link"><a href="#homepage">Home page!</a></li>
+          <li id="aboutUs-link"><a href="#about">About Us</a></li>
+          <li id="animals-link"><a href="#animal">Endangered Animals</a></li>
+          <li id="adoptPet-link"><a href="#adoptPet">Adopt A Pet</a></li>
+          <li id="surrenderPet-link"><a href="#surrenderPet">Surrender My Pet</a></li>
```

Comment:
Do these links bring you anywhere in your app? Another way we could approach a navbar is to create a React component that lists all of these available links, and then we can render the navbar on each page from the App.js component - which is where we will also want to define our BrowserRouter. We want to make these links available in our React app so that React can help direct the user to the correct component, rather than processing an HTTP request each time

Reply:
Thanks for the feedback, I'll make the edits are re-submit the PR!